security-policies/bundle/compliance/cis_azure/rules/cis_4_3_8/rule.rego

```diff
@@ -0,0 +1,22 @@
+package compliance.cis_azure.rules.cis_4_3_8
+
+import data.compliance.lib.common
+import data.compliance.policy.azure.data_adapter
+import future.keywords.if
+
+finding = result if {
+	# filter
+	data_adapter.is_postgresql_single_server_db
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(infrastructure_encryption_enabled),
+		{"Resource": data_adapter.resource},
+	)
+}
+
+default infrastructure_encryption_enabled = false
+
+infrastructure_encryption_enabled if {
+	data_adapter.properties.infrastructureEncryption == "Enabled"
```

Comment:
I see that we couldn't test it since we can't create postgre single servers, but is it possible to verify the `Enabled` will be capitalized? I there any chance it will be lower case in some cases?

Reply:
The capital letter is the pattern from Azure Assets Resource Graph. [You can see a implementation with the properties from ARG here](https://github.com/elastic/cloudbeat/blob/main/security-policies/bundle/compliance/cis_azure/rules/cis_4_3_1/rule.rego).

So it _should_ be with capital. With that said, I agree that it's sensible and if azure changes its pattern we would have false positives. 

Just for safety I will lower case it and check with `enabled` instead of `Enabled`. Sounds good?

Reply:
https://github.com/elastic/cloudbeat/pull/1797/files#diff-b2e6d792dfee3d7ee031452d839f317586707fa0c7c9479ef69189b4340106e9R21 lower cased it and added tests on it